resources/views/verify.blade.php

```diff
@@ -0,0 +1,12 @@
+@extends('layouts.master', ['bodyClass' => 'auth'])
+
+@section('content')
+    @php
+        use \Illuminate\Support\Facades\Cache;
+        $key = Illuminate\Support\Facades\Crypt::decrypt(request('token'));
+        $identifier = Cache::get($key);
+        $code = Cache::get('otp-'.$identifier);
+    @endphp
```

Comment:
extract to controller

Reply:
this is temporary, used only to display the otp on the page.